ocs_ci/deployment/deployment.py

```diff
@@ -1990,6 +1993,11 @@ def deploy_with_external_mode(self):
         logger.info("Checking ceph health for external cluster")
         if not config.DEPLOYMENT.get("multi_storagecluster"):
             try:
+                res = wait_for_ceph_cmd_execute_successfully(timeout=120, sleep=10)
+                if not res:
+                    logger.info("Trying to restart the rook-ceph-operator pod...")
+                    operator_pods = get_operator_pods()
+                    delete_pods(operator_pods)
```

Comment:
why do we need to restart operator pod here since its new deployment

Reply:
This is a problem I observed recently with the external mode 4.18 cluster. I created an issue regarding the matter: https://github.com/red-hat-storage/ocs-ci/issues/11538

Reply:
do we have jira for the bug?

Reply:
@vavuthu this should not impact the customer environment as in external mode we wont be shipping the toolbox pod, so we just to have a workaround for ocs ci